src/app/series/page.tsx

```
@@ -0,0 +1,100 @@
+"use client";
+
+import { Box, Container, Stack } from "@mui/material";
```

ESLint

[error] 'Container' is defined but never used.

(@typescript-eslint/no-unused-vars)